pylossless/dash/qcgui.py

```diff
@@ -135,7 +135,7 @@ def set_visualizers(self):
 
     def update_bad_ics(self):
         """Add IC name to raw.info['bads'] after selection by user in app."""
-        df = self.pipeline.flagged_ics.data_frame
+        df = self.pipeline.flags['ic'].data_frame
```

Comment:
This needed to be updated after our flags class refactor so the dashboard can still launch...